cloud/google/node_init.sh

```diff
@@ -6,4 +6,21 @@ sudo mkdir ${DATA_DIR}
 sudo /usr/share/google/safe_format_and_mount \
   -m "mkfs.ext4 -F" ${DATA_DEV} ${DATA_DIR}
 
-
+# Install google-fluentd which pushes application log files up into the Google
+# Cloud Logs Monitor.
+curl -sSO https://storage.googleapis.com/signals-agents/logging/google-fluentd-install.sh
+sha256sum ./google-fluentd-install.sh
```

Comment:
The bit in the docs where it talking about doing this via the start-up script (as I've done) don't verify the sha256, I guess because you'd have to keep updating the script as they pushed new versions.  so I took the approach of logging the sha256 so at least you could look back and see.

I don't really mind either way.